spec.html

```diff
@@ -37838,33 +37841,15 @@ <h1>AtomicReadModifyWrite ( _typedArray_, _index_, _value_, _op_ )</h1>
         <p>The abstract operation AtomicReadModifyWrite takes arguments _typedArray_, _index_, _value_, and _op_ (a read-modify-write modification function). _op_ takes two List of byte values arguments and returns a List of byte values. This operation atomically loads a value, combines it with another value, and stores the result of the combination. It returns the loaded value. It performs the following steps when called:</p>
         <emu-alg>
           1. Let _buffer_ be ? ValidateSharedIntegerTypedArray(_typedArray_).
-          1. Let _i_ be ? ValidateAtomicAccess(_typedArray_, _index_).
+          1. Let _indexedPosition_ be ? ValidateAtomicAccess(_typedArray_, _index_).
           1. Let _arrayTypeName_ be _typedArray_.[[TypedArrayName]].
           1. If _typedArray_.[[ContentType]] is ~BigInt~, let _v_ be ? ToBigInt(_value_).
           1. Otherwise, let _v_ be ? ToInteger(_value_).
-          1. Let _elementSize_ be the Element Size value specified in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _arrayTypeName_.
           1. Let _elementType_ be the Element Type value in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _arrayTypeName_.
-          1. Let _offset_ be _typedArray_.[[ByteOffset]].
-          1. Let _indexedPosition_ be (_i_ &times; _elementSize_) + _offset_.
           1. Return GetModifySetValueInBuffer(_buffer_, _indexedPosition_, _elementType_, _v_, _op_).
         </emu-alg>
       </emu-clause>
 
-      <emu-clause id="sec-atomicload" aoid="AtomicLoad">
```

Comment:
Removing this will un-define the id "sec-atomicload". Should it become an oldid somewhere?

Reply:
I think it's moderately unlikely that anyone will have linked it, and I think it'll be pretty easy to figure out what to look at instead of so, so I'm not too worried about it.

Reply:
I'd prefer to always keep oldids if there's a reasonable place to link them; crawlers will have linked it.

Reply:
There is no corresponding abstract operation anymore because this operation is getting removed entirely. `Atomics.load` is the closest thing, but it's a built-in function, not an AO, so it's a little weird to tag it.

I don't really care if crawlers will have linked it if no human is ever going to click on those links, and I don't really expect a human to ever click on a such a crawled link.

Reply:
URLs should all work forever :-) but if there's no sensible place to link it, then it sounds right not to use an oldid.

Reply:
Basically it comes down to whether or not we want to add it to `Atomics.load`, which does the same thing (prior to this PR its definition was just "call AtomicLoad")  but is a built-in function instead of an abstract operation. I'd lean towards no, but I don't feel strongly.

Reply:
Yeah why not we can add an oldid.

Reply:
Done.